src/components/color-picker/color-picker.component.ts

```diff
@@ -106,6 +106,10 @@ export class ColorPickerComponent implements OnInit, OnDestroy {
     @Input()
     inputAriaLabel: (mode: ColorPickerInputMode) => string = this.getInputAriaLabel;
 
+    /** Whether or not to forward focus to the selected or first button in the color picker. */
+    @Input()
+    forwardFocusToButton: boolean = false;
```

Comment:
I think this should be the default behavior, no need for configuration. I don't see any need for landing focus on the wrapper.